pkg/stores/sqlpartition/listprocessor/processor.go

```diff
@@ -54,6 +69,48 @@ type Cache interface {
 	ListByOptions(ctx context.Context, lo informer.ListOptions, partitions []partition.Partition, namespace string) (*unstructured.UnstructuredList, int, string, error)
 }
 
+func k8sOpToRancherOp(k8sOp selection.Operator) (informer.Op, bool, error) {
+	v, ok := mapK8sOpToRancherOp[k8sOp]
+	if ok {
+		return v, k8sOp == selection.PartialEquals || k8sOp == selection.NotPartialEquals, nil
+	}
+	return "", false, fmt.Errorf("unknown k8sOp: %s", k8sOp)
+}
+
+// Determine if the value field is surrounded by a pair of single- or double-quotes
+// This is a difference we implement from the kubernetes CLI: if the target value of a (not) equal
+// test is single-quoted, we use the full string. Otherwise we do a substring match
+// (which is implemented as 'SELECT ... some-field ... LIKE "%VALUE%" ...' in the query)
+//
+// The caller also needs to know if it should strip delimiting quotes, so this returns two bools
+func isQuotedStringTarget(values []string) (isQuoted bool, isSingleQuoted bool) {
```

Comment:
IIUC this is dead code now, can we drop it?

Reply:
dropped